recipes/openfoam/meta.yaml

```diff
@@ -0,0 +1,71 @@
+{% set name = "openfoam" %}
+{% set version = "v2412" %}
+{% set build = 0 %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://develop.openfoam.com/Development/openfoam/-/archive/OpenFOAM-{{ version }}/openfoam-OpenFOAM-{{ version }}.tar.gz
+  sha256: 27d06edb4a6ffc6f88a494edd3b289d5d0a0874866a416b6b0d0a26aba2d9c4a
+
+build:
+  skip: True  # [not linux]
+  number: {{ build }}
+  rpaths:
+    - lib/
+    - lib/sys-mpich/
+    - lib/dummy/
+
+requirements:
+  build:
+    - gcc
+    - gxx
```

Comment:
Please use the compiler() functions instead of listening gcc and gxx here.

Also, all library dependencies should be in host, not build.

Reply:
Hi, 
I did my best to provide the requested changes. The pipelines seems to fail on timeout, since this is such a big piece of software (but updated only once per year or so), could this project get some more time to build ?

Thank you in advance for your review,